typeclasses/armors.py

```diff
@@ -2,11 +2,90 @@
 Armor typeclasses
 """
 
-from typeclasses.items import Item
+from typeclasses.items import Equippable
+
+class Armor(Equippable):
+    """Typeclass for armor objects.
+
+    Attributes:
+      toughness (int): primary defensive stat
+    """
+    toughness = 0
 
-class Armor(Item):
     def at_object_creation(self):
         super(Armor, self).at_object_creation()
-        self.toughness = 0
-        self.abilities = []
-        self.slots = []
+        self.db.toughness = self.toughness
+
+
+# typeclasses for prototyping
+
+
+class Headwear(Armor):
+    """Typeclass for armor prototypes worn on the head."""
+    slots = ['head']
+
+
+class Shoulderwear(Armor):
+    """Typeclass for armor prototypes worn on the shoulders."""
+    slots = ['shoulders']
+
+
+class Breastplate(Armor):
+    """Typeclass for sleeveless armor prototypes."""
+    slots = ['torso']
+
+
+class UnderShirt():
+    """Typeclass for shirts worn beneath Breastplates."""
+    slots = ['arms']
+
+
+class SleevedShirt(Armor):
```

Comment:
I think it's desirable to wear a breastplate and shirt at the same time, so we need to resolve the conflict of both in a torso slot if the equip code doesn't handle layering.